ote/src/clj/ote/services/service_search.sql

```diff
@@ -1,8 +1,9 @@
 -- name: operation-area-facet
 SELECT "operation-area" as text, COUNT(*) as count
-  FROM "operation-area-facet"
+  FROM "operation-area-facet" o
+ WHERE o."operation-area" IN (SELECT f.namefin FROM "finnish_municipalities" f)
```

Comment:
should we instead do this when the data is generated in `transport_service_operation_area_array` trigger and recreate the full data in a migration